web/js/tablesorter.js

```diff
@@ -2,7 +2,7 @@ $(document).ready(function() {
 
     // call the tablesorter plugin and apply the uitheme widget
     $("table").tablesorter({
-        theme : "bootstrap",
+        theme : "default", /* or "bootstrap" */
```

Comment:
Wann or?

Reply:
hardcoded, damit man weiß, dass bootstrap die Alternative zu default ist.
dark theme im tablesorter header vs. light theme

Reply:
Wollen wir den Kommentar dann noch etwas genauer machen. Dann fände ich das sehr sinnvoll.

Reply:
Heute ist mein Kopf etwas konfus. Hast du einen Vorschlag?

Reply:
Klar: "use 'bootstrap' for a grey header"
Das ist doch der wichtige Unterschied oder?